### ISSUE TITLE
Sférické fotky

### ISSUE THREAD

**Issue by kalich5** [skaut/skaut-google-drive-gallery#97]
Máme žádost o přidání možnosti sférických fotek. 

@GenaBitu Je to reálné, podle mě by v tom neměl být problém.

- s tím se pojí i to, že by u takových fotek měl být nějaký piktogram zeměkoule (jako u jiných služeb)

**Comment by kalich5** [comment]
https://photos.google.com/share/AF1QipOkUEVg67R7nnhYPQwV6WEQiQ1ie4eYN2s-wiK1Tiaubaigv84npccrfmWzunDRag?key=cFRCNzlQMkJTNUFhLWlON2tEUWw3RnVhSU5QU1ln

**Comment by kalich5** [comment]
Odkaz na fotku na testování :-)

**Comment by GenaBitu** [comment]
Kdo žádá? :D

Jo, to je podle mě přesně typ funkčnosti, kterou můžeme přidat - kdo to nepoužívá, ani o tom neví a nijak ho to nelimituje, kdo to používá, je to tam.

Ale na druhou stranu, v pořadí priorit je podle mě tohle hóódně hluboko.

**Comment by kalich5** [comment]
Na skautfóru o tom psal jeden kluk. A přijde mi to, jako věc co bych i já na webu využil. Mám z letošního tábora sférické fotky a nemám je teď jak zobrazit...

**Comment by GenaBitu** [comment]
Jo, podpora sférických fotek existující uživatele nic nestojí, takže za mě dobrá featura. Otevřeno PR, za mě je to v seznamu priorit poměrně nízko.
